### ISSUE TITLE
Hybrid Attributes and TextFields don't properly form an expression (SQLite)

### ISSUE THREAD

**Issue by ballan91** [coleifer/peewee#1241]
Basic repro below, but just as the description says, I cannot seem to get the two to play nicely, I was going off some examples in SQLAlchemy which I believe was the basis for hybrid attributes.

Using SQLite and tested with the latest release version of peewee 2.9.1

```python
from peewee import TextField, Model
from playhouse.hybrid import hybrid_property


class TestModel(Model):

    first_name = TextField()
    last_name = TextField()

    @hybrid_property
    def full_name(self):

        return self.first_name + ' ' + self.last_name

def test():

    TestModel.create_table(fail_silently=True)

    TestModel.insert(
        first_name='John',
        last_name='Doe'
    ).execute()

    for person in TestModel.select():
        print(person.full_name)

    q = TestModel.select().where(
        TestModel.full_name == 'John Doe'
    )

    if q.count() == 0:
        print('Failed with')
        print(q.sql())
    else:
        print('Success')

if __name__ == '__main__':
    test()
```

**Comment by coleifer** [comment]
Cool -- I think I need to modify the string based fields to use concatenation for the `+` operator overload. Nice catch.

**Comment by coleifer** [comment]
1740cc1132ca01d3964f543eca00f1dc9532bc24

**Comment by ballan91** [comment]
Thanks for such a swift reply! Release schedule wise, there appear to be a number of fixes since the last release, do you expect to issue a new release in the near future?

**Comment by coleifer** [comment]
Just released 2.9.2, thanks!
